modules/upgrade-upgrading-cli.adoc

```diff
@@ -0,0 +1,147 @@
+* Log in to the cluster as user with `cluster-admin` privileges.
+* Install the `jq` package.
+
+.Procedure
+
+. Ensure that your cluster is available:
++
+----
+$ oc get clusterversion
+
+NAME      VERSION     AVAILABLE   PROGRESSING   SINCE     STATUS
+version   4.1.0-rc.4   True        False         24h       Cluster version is 4.1.0-rc.4
+----
++
+If your `VERSION` is `4.1.0-rc.4`, you installed OpenShift 4 Beta Drop 5, and
+you can update to {product-title} {product-version}.
+
+. Review the current update channel information and confirm that your channel
+is set to `stable-4.1`:
++
+----
+$ oc get clusterversion -o json|jq ".items[0].spec"
+
+{
+  "channel": "stable-4.1",
+  "clusterID": "53f97452-9856-4a3c-8260-00c1de2668a1",
+  "upstream": "https://api.openshift.com/api/upgrades_info/v1/graph"
+}
+----
+
+. View the available updates and note the version number of the update that
+you want to apply:
++
+----
+$ oc adm upgrade
+
+Cluster version is 4.0.0-0.nightly-2019-01-25-200832
+Updates:
+VERSION                       	IMAGE
+4.1.0-0.nightly-2019-01-25-201056 registry.svc.ci.openshift.org/ocp/release:4.1.0-0.nightly-2019-05-25-201056
+4.1.0-0.nightly-2019-01-25-205123 registry.svc.ci.openshift.org/ocp/release:4.1.0-0.nightly-2019-05-25-205123
+----
+
+. Apply an update:
+** To update to the latest version:
++
+----
+$ oc adm upgrade --to-latest=true <1>
+----
+
+** To update to a specific version:
++
+----
+$ oc adm upgrade --to=<version> <1>
+----
+<1> `<version>` is the update version that you obtained from the output of the
+previous command.
+
+. Review the status of the Cluster Version Operator:
++
+----
+$ oc get clusterversion -o json|jq ".items[0].spec"
+
+{
+  "channel": "stable-4.1",
+  "clusterID": "53f97452-9856-4a3c-8260-00c1de2668a1",
+  "desiredUpdate": {
+	"image": "registry.svc.ci.openshift.org/ocp/release:4.1.0-0.nightly-2019-05-25-201056",
+	"version": "4.1.0-0.nightly-2019-05-25-201056" <1>
+  },
+  upstream": "https://api.openshift.com/api/upgrades_info/v1/graph"
+}
+----
+<1> If the `version` number in the `desiredUpdate` stanza matches the value that
+you specified, the update is in progress.
+
+. Review the cluster version status history to monitor the status of the update.
+It might take some time for all the objects to finish updating.
++
+----
+$ oc get clusterversion -o json|jq ".items[0].status.history"
+
+[
+  {
+	"completionTime": null,
+	"image": "registry.svc.ci.openshift.org/ocp/release:4.1.0-0.nightly-2019-05-25-201056",
+	"startedTime": "2019-01-29T07:51:34Z",
+	"state": "Partial",
+	"version": "4.1.0-0.nightly-2019-05-25-201056"
+  },
+  {
+    "completionTime": "2019-05-20T15:47:21Z",
+    "image": "registry.svc.ci.openshift.org/ocp/ocp-release@sha256:6f4cf2db7e63c4dba54496a72b83fec22c49293b520ff0cdb78f1e38b23f1ccb",
+    "startedTime": "2019-05-20T15:23:29Z",
+    "state": "Completed",
+    "verified": false,
+    "version": "4.1.0-rc.4"
+  }
+]
+----
```

Comment:
Refer to https://github.com/openshift/openshift-docs/pull/13993/files#r286777865